api/activity-group/models/activity-group.js

```diff
@@ -1,8 +1,23 @@
-'use strict';
+"use strict";
+const {
+  deleteFromAlgolia,
+  updateInAlgolia,
+} = require("../../../utils/algolia");
 
 /**
  * Read the documentation (https://strapi.io/documentation/developer-docs/latest/development/backend-customization.html#lifecycle-hooks)
  * to customize this model
  */
 
-module.exports = {};
+const contentType = "activity-group";
+
+module.exports = {
+  lifecycles: {
+    afterUpdate(result, params, data) {
+      updateInAlgolia(contentType, result);
+    },
+    afterDelete(result, params) {
+      deleteFromAlgolia(contentType, result.id);
```

Comment:
It might 🤔 . Only in the case when you would need to add something else to a content type's lifecycle hooks (which these `afterDelete` and such are), it would need some code. 

I'll make an improvement and lets see.

Reply:
How about now?